src/main/java/com/iota/iri/Iota.java

```diff
@@ -279,6 +291,20 @@ private void initializeTangle() {
         }
     }
 
+    private void injectLocalSnapshotsDb() throws Exception{
+        localSnapshotsDb.injectProvider(createRocksDbProvider(
+                configuration.getLocalSnapshotsDbPath(),
+                configuration.getLocalSnapshotsDbLogPath(),
+                configuration.getDbConfigFile(),
+                1000,
+                new HashMap<String, Class<? extends Persistable>>() {{
+                    put("spent-addresses", SpentAddress.class);
+                    put("localsnapshots", LocalSnapshot.class);
+                }}, null));
```

Comment:
Can't all of this (maybe besides the init()) be part of `MainnetInjectionConfiguration`?

Reply:
It was there originally but Luca had moved it out previously so I thought putting it where the tangle db creation happened would be a good place.